test/simulation.jl

```diff
@@ -451,6 +451,62 @@ end
     @test maximum(maximum(abs.(v)) for v in (s1.coords .- s2.coords)) < 1e-5u"nm"
 end
 
+@testset "Temperature Replica Exchange" begin
+    n_atoms = 100
+    atom_mass = 10.0u"u"
+    atoms = [Atom(mass=atom_mass, σ=0.3u"nm", ϵ=0.2u"kJ * mol^-1") for i in 1:n_atoms]
+    boundary = CubicBoundary(2.0u"nm", 2.0u"nm", 2.0u"nm")
+    coords = place_atoms(n_atoms, boundary, 0.3u"nm")
+
+    pairwise_inters = (LennardJones(),)
```

Comment:
`nl_only=true` or it won't use the neighbour list (also in the other test file).